Dockerfile

```diff
@@ -1,8 +1,17 @@
-FROM yunspace/alpine-golang
+FROM golang:alpine
 
-ADD build/codeclimate-gofmt /usr/src/app/
+MAINTAINER hello@codeclimate.com
 
 RUN adduser -u 9000 -D app
+
+WORKDIR /code
+
+COPY build/ /usr/src/app
+COPY build/codeclimate-gofmt /usr/src/app/build/codeclimate-gofmt
+RUN chown -R app:app /usr/src/app
```

Comment:
It does. Got rid of that and also the `USER app` line which isn't in the spec example and wasn't needed for engine to work locally.